### PR TITLE
Update to Zewo 0.14

### PR DESCRIPTION
**Pull request by Danappelxx**
Passes all tests. LGTM, but should still be reviewed before merging.

### CODE REVIEW

File: Sources/TodoBackend/Models/Entity.swift [hunk 1]
```diff
@@ -36,19 +12,19 @@ struct Entity<Item: StructuredDataConvertible> {
     }
 }
 
-extension Entity: StructuredDataConvertible {
-    var structuredData: StructuredData {
-        var dict = item.structuredData.dictionaryValue!
-        dict["id"] = .infer(id)
-        dict["url"] = .infer(url)
-        return .infer(dict)
+extension Entity : MapInitializable, MapRepresentable {
```
Comment (paulofaria) [inline Sources/TodoBackend/Models/Entity.swift]: This can be `Entity : MapConvertible`

File: Sources/TodoBackend/Models/Todo.swift [hunk 2]
```diff
@@ -33,37 +8,36 @@ struct Todo {
...
+            title: map["title"].string,
+            completed: map["completed"].bool,
+            order: map["order"].int
         )
     }
 }
 
-extension Todo: ContentMappable, StructuredDataConvertible {
-    init(mapper: Mapper) throws {
+extension Todo : MapInitializable, MapRepresentable {
```
Comment (paulofaria) [inline Sources/TodoBackend/Models/Todo.swift]: `MapConvertible`

**Review by paulofaria — APPROVED** [review]
LGTM.. we could not implement the property and initializer required by `MapConvertible`. and let it use reflection, but that would make it slower. If were sending this to todobackend.io this way it's fine.

**Comment by Danappelxx** [comment]
Good idea - the throwableness of `asMap()` actually removes a force-unwrap in this case which is very nice. 

I'll use reflection for the representation of `Todo`, but the initializer has defaults and `Entity`'s serialization/representation is inline rather than nested so it has to be done manually as well. I don't mind the performance hit since it cleans things up.

Anyway, updated. Merge when ready :)

**Pull request merged**